### ISSUE TITLE
// eslint-disable ... comments are not ignored anymore by line-comment-position rule

### ISSUE THREAD

**Issue by chrisguttandin** [eslint/eslint#11415]
**Tell us about your environment**

* **ESLint Version:** 5.14.1
* **Node Version:** 11.10.0
* **npm Version:** 6.8.0

**What parser (default, Babel-ESLint, etc.) are you using?** default (parserOptions: {  ecmaVersion: '2019', sourceType: 'module' })

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```json
{
    "env": {
        "es6": true
    },
    "parserOptions": {
        "ecmaVersion": "2019",
        "sourceType": "module"
    },
    "rules": {
        "no-bitwise": "error",
        "line-comment-position": [ "error", { "position": "above" } ]
    }
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

When linting a file which uses in-line comments to disable a certain rule for only one line I get an error with the configuration above in eslint@5.14.x

```js
const x = 20;

console.log(x >> 5); // eslint-disable-line no-bitwise
```

```bash
eslint -c config.json script.js 
```

**What did you expect to happen?**
With previous versions of eslint nothing happens. // eslint-... comments where ignored by the line-comment-position rule.

**What actually happened? Please include the actual, raw output from ESLint.**
I get an error "3:22  error  Expected comment to be above code  line-comment-position".

**Are you willing to submit a pull request to fix this bug?**
Generally yes, but I have no idea where to start.

**Comment by platinumazure** [comment]
Hi @chrisguttandin, thanks for the issue. This does seem like a bug.

By any chance, do you know if this was working in 5.13.x?

**Comment by chrisguttandin** [comment]
Hi @platinumazure, thanks for the quick response. I can confirm, that it was working with version 5.13.x.

**Comment by not-an-aardvark** [comment]
I can reproduce this issue. Based on `git bisect`, it was caused by 550de1e611a1e9af873bcb18d74cf2056e8d2e1b.
